test/integration/an_item_can_be_removed_from_cart_test.rb

```diff
@@ -15,4 +15,39 @@ class AnItemCanBeRemovedFromCartTest < ActionDispatch::IntegrationTest
     refute page.has_content?("Gnar possum")
     refute page.has_content?("1000")
   end
+
+  test 'user is redirected to /cart after removing an item' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.